Sources/RInAppMessaging/RInAppMessaging.swift

```diff
@@ -211,6 +216,30 @@ import RSDKUtils
         return configURL
     }
 
+    /// Checks the existence of RMC module and verifies the `configure()` caller.
+    /// - Parameter subscriptionID: a subscriptionID value from `configure()` call to check for '-rmc' suffix
+    /// - Returns: `false` if RMC module is integrated and the method wasn't called from the RMC module.
+    internal static func verifyRMCEnvironment(subscriptionID: String?) -> Bool {
+        guard isRMCEnvironment else {
+            return true
+        }
+
+        return subscriptionID?.hasSuffix(Constants.RMC.subscriptionIDSuffix) == true
```

Comment:
can we use `nil coalescing` like `?? false?

Reply:
@Esakkiraja-Pothikannan But how? `(subscriptionID ?? "").hasSuffix(Constants.RMC.subscriptionIDSuffix) == true`?
Only `subscriptionID` is optional here

Reply:
I meant this way `subscriptionID?.hasSuffix(Constants.RMC.subscriptionIDSuffix) ?? false`

Reply:
Ah ok. Interesting. Not sure which looks better. I think mine is more straightforward to be honest.